Remove `prismaClient.$on('beforeExit'...)` to assist with Prisma Data Proxy support (#8347)
keystonejs/keystone: .changeset/strange-baboons-do.md

---
'@keystone-6/core': patch
---

> Removes `prismaClient.$on('beforeExit'...` as it is no longer required and blocked Prisma Data Proxy support

Reviewer: ```suggestion
Removes outdated Prisma `prismaClient.$on('beforeExit'...` work-arounds that blocked Prisma Data Proxy support
```
